### PR TITLE
kolkman2022strategic

### PR DESCRIPTION
**Pull request by GuillyK**
## Project Name

kolkman2022strategic

## Compiled PDF

Add the link to the compiled pdf here. Example : [Compiled Pdf](https://github.com/GuillyK/MLRC/blob/main/2021/kolkman2022strategic/journal/article.pdf)
## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

We were able to compile our pdf now. However, our citations and references are not correct. And the first page looks scrambled. I hope you can help us.
```
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl in line'
 No file article.bbl.
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation 'credit_fraud' on page 7 undefined on input line 203
  Citation 'financial_distress' on page 7 undefined on input line 202
  Citation 'hardt2016strategic' on page 5 undefined on input line 149
  Citation 'hardt2016strategic' on page 6 undefined on input line 174
  Citation 'hardt2016strategic' on page 7 undefined on input line 201
  Citation 'hardt2016strategic' on page 9 undefined on input line 256
  Citation 'hardt2016strategic' on page 9 undefined on input line 287
  Citation 'hardt2016strategic' on page 9 undefined on input line 288
  Citation 'levanon2021strategic' on page 2 undefined on input line 9
  Citation 'levanon2021strategic' on page 3 undefined on input line 71
  Citation 'levanon2021strategic' on page 5 undefined on input line 115
  Citation 'levanon2021strategic' on page 5 undefined on input line 136
  Citation 'spam_data' on page 7 undefined on input line 201
  Citation 'ustun2019actionable' on page 7 undefined on input line 204
  Reference `eq:SERM' on page 12 undefined on input line 344
  Reference `eq:SERM' on page 5 undefined on input line 145
  Reference `eq:resp_map' on page 12 undefined on input line 344
  Reference `fig:confusion' on page 10 undefined on input line 306
  Reference `fig:extention_results' on page 10 undefined on input line 291
  Reference `fig:semi-strat' on page 10 undefined on input line 306
  Reference `sec:datasets' on page 6 undefined on input line 166
  Reference `sec:model_desc' on page 8 undefined on input line 220
  Reference `sec:scope' on page 7 undefined on input line 192
  Reference `tab:accuracies' on page 9 undefined on input line 287
  Reference `tab:accuracies' on page 9 undefined on input line 288
  Reference `tab:datasets' on page 7 undefined on input line 200
  Reference `tab:hyperparams_reg' on page 8 undefined on input line 220
  Reference `tab:hyperparams_vanilla' on page 8 undefined on input line 220
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target `article.bbl'.  Stop.
Latexmk: Summary of warnings from last run of *latex:
  Latex failed to resolve 15 reference(s)
  Latex failed to resolve 16 citation(s)
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'article.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
make: *** [article.pdf] Error 12
Build complete
rm: article.bbl: No such file or directory
rm: article.blg: No such file or directory
```
If your build had any errors locally, please add it here. You can upload your build.log in a [Pastebin](https://pastebin.com/) link and put it here.

### CODE REVIEW

**Comment by SharathRaparthy** [comment]
@GuillyK could you please look into table/figure formatting? Looks like figure 1 is not fitting?

**Comment by GuillyK** [comment]
Figure 1 has the correct formatting now. However, we still don't know how to solve the citations/references

File: 2021/kolkman2022strategic/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,20 @@
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{dsfont}
+\usepackage{multirow}
+ \usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline 2021/kolkman2022strategic/openreview/packages.tex:10]: Please remove xcolor package (reasoning in README)

**Comment by koustuvsinha** [comment]
Hi, your compiled pdf (after removing xcolor) contains reproducibility summary thrice in the first two pages. Please remove them and start with the reproducibility summary.

**Comment by GuillyK** [comment]
We have removed the reproducibility summary duplication. However, the title page isn't correct. The side column with "edited by..." Has overlap with the first page. Our content.tex file seems to have the same starting structure as accepted pull requests. This is also the same case for the [main.tex]. (https://github.com/GuillyK/MLRC/blob/main/2021/kolkman2022strategic/openreview/main.tex)
Lastly, the bibliography still does not show up.

```
\section{ Reproducibility Summary}


\subsection{Scope of Reproducibility}

In this work, the paper Strategic Classification Made Practical\cite{levanon2021strategic} is evaluated through a reproduction study. The results from the reproduction examines if the claims made in the paper are valid. We could find two main claims that were made by the authors that we will attempt to reproduce. Those are as follows:

% The claims made in the paper that will be reproduced are as follows:
\begin{enumerate}
    \item "We propose a novel learning framework for strategic classification that is practical, effective, and flexible. This allows for differentiation through strategic user responses, which supports end-to-end training."
    \item "We propose several forms of regularization that encourage learned models to promote favorable social outcomes."
\end{enumerate}

We interpret \textit{practical, effective and flexible} as such that the model should work better on a variety of real life problems than their non-strategic counterpart.
 
```

**Comment by SharathRaparthy** [comment]
Hello @GuillyK , I just compiled it on my mac and the generated pdf looks good to me (and can see the references as well). So currently marking this PR as latex-ok. Let me know if you are still facing difficulties with the bibliography.

**Comment by GuillyK** [comment]
@SharathRaparthy Does this mean you can also see the bibliography in the pdf? If that is the case then it is complete from what we needed to do/add. There is then something wrong locally with my mac that the pdf doesn't generate well. 

This is how my pdf looks. Thus, I'm glad that you can generate the good pdf.
Screenshot 2022-05-10 at 21 51 52

**Comment by SharathRaparthy** [comment]
@GuillyK Yes, this is how the report looks after I compiled it.
Screen Shot 2022-05-10 at 11 09 35 PM


Moreover, I just noticed that the title ("A report in MLRC 2021") is incorrect. Could you just change the title?

**Comment by GuillyK** [comment]
@SharathRaparthy Hi Sharath, I wanted to check if everything works now and is ready to be published since the update requested tag is still there after changing the title.

**Pull request merged**